azure/image/misc-settings.sh

```diff
@@ -131,6 +135,13 @@ TLS_OPTIONS=-cert-file /etc/certificates/tls.crt -cert-key /etc/certificates/tls
 END
 fi
 
+mkdir -p /etc/systemd/system/attestation-agent.service.d
+cat <<END >/etc/systemd/system/attestation-agent.service.d/10-grpc.conf
+[Service]
+ExecStart=
+ExecStart=/usr/local/bin/attestation-agent.grpc --getresource_sock 127.0.0.1:50001
+END
+
```

Comment:
I'll move this to a separate PR but this is kind of a policy question. If none of the CAA users have a use for ttRPC I'll switch it to gRPC and make sure we have two.

For sure `/usr/local/bin/attestation-agent` needs to stay ttrpc so that kata-agent works (can be fixed but that's how it is right now).

Reply:
> If none of the CAA users have a use for ttRPC

The current unit has never worked with ttRPC, as it uses an IP address as socket and will always fail (https://github.com/confidential-containers/cloud-api-adaptor/issues/1015). So I assume it isn't used by anyone.